app/controllers/api/v1/comments_controller.rb

```diff
@@ -7,10 +7,16 @@ class CommentsController < ApplicationController
       before_action :set_comment, only: %i[update]
 
       def index
-        @comment = Comment.order('created_at DESC')
-        render json: CommentsSerializer.new(@comment,
-                                            fields: { comments: :body })
-                                       .serializable_hash, status: :ok
+        if params[:news_id]
+          @news = News.find(params[:news_id])
+          @comment = @news.comments
+          render json: CommentsSerializer.new(@comment).serializable_hash
```

Comment:
Please check this:
```suggestion
          render json: CommentSerializer.new(@comment).serializable_hash
```

Reply:
done, also i changed @comment to @comments